ksql-parser/src/main/java/io/confluent/ksql/parser/SqlFormatter.java

```diff
@@ -410,6 +426,55 @@ protected Void visitDropTable(final DropTable node, final Integer context) {
       return null;
     }
 
+    @Override
+    protected Void visitTerminateQuery(final TerminateQuery node, final Integer context) {
+      builder.append("TERMINATE ");
+      builder.append(node.getQueryId().getId());
+      return null;
+    }
+
+    @Override
+    protected Void visitListTables(final ListTables node, final Integer context) {
+      builder.append("SHOW TABLES");
+      if (node.getShowExtended()) {
+        visitExtended();
+      }
+      return null;
+    }
+
+    @Override
+    protected Void visitListStreams(final ListStreams node, final Integer context) {
+      builder.append("SHOW STREAMS");
+      if (node.getShowExtended()) {
+        visitExtended();
+      }
+      return null;
+    }
+
+    @Override
+    protected Void visitUnsetProperty(final UnsetProperty node, final Integer context) {
+      builder.append("UNSET '");
+      builder.append(node.getPropertyName());
+      builder.append("'");
+
+      return null;
+    }
+
+    @Override
+    protected Void visitSetProperty(final SetProperty node, final Integer context) {
```

Comment:
@agavra I had to support SET and UNSET here because the `KsqlEngineTest` tests were failing (shouldSetPropertyInRunScript and shouldUnsetPropertyInRunScript).

But now the `ensure-formatter.json` is failing because SET is supported. 
- Having SET supported in SqlFormatter is good? 
- Should I remove the two test cases above (they're not in master)
- Or, should I add a different statement in the ensure-formatter that fails?

Reply:
Hmmm... SET support is a good thing. I just couldn't think of another good way for `ensure-formatter` to work. I'm happy to get suggestions. (I don't think we should remove the existing tests because 5.3. depends on that). One option is to remove `ensure-formatter` for this PR

Reply:
I removed it so I can move forward.